src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
I think we can simplify variable names by removing part `ball`. It would be greater to write `ball.color`  instead of `ball.ballColor`. Please, replace it with 
```
 private String color;
 private int number;

```